src/System.Management.Automation/CoreCLR/CorePsExtensions.cs

```diff
@@ -1232,7 +1232,7 @@ private static class Win32Native
                                                      int dwLanguageId, [Out]StringBuilder lpBuffer,
                                                      int nSize, IntPtr va_list_arguments);
 
-            [DllImport("api-ms-win-core-sysinfo-l1-2-1.dll", CharSet = CharSet.Unicode, SetLastError = true)]
+            [DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
>32.dll", [](start = 30, length = 8)

Let us use same strategy we are planning on using for PInvokeDllNames.cs file instead of removing it.